Python/omevv/v1/scripts/CreateRepoProfile.py

```diff
@@ -0,0 +1,122 @@
+import argparse
+import sys
+import time
+import warnings
+from webbrowser import get
+
+import requests,json,base64
+warnings.filterwarnings("ignore")
+
+parser = argparse.ArgumentParser(
```

Comment:
Please do not use ArgParser as a global construct. This will prevent the script being used as a module. Instead use it within your main function